vignettes/GGIR.Rmd

```diff
@@ -151,358 +259,542 @@ GGIR(datadir="C:/mystudy/mydata",
+
+### Key arguments related to sleep analysis {#key-arguments-related-to-sleep-analysis}
+
+For an explanation on how sleep is detected and the specific role of the
+various function arguments see section [Sleep
+analysis](#Sleep_analysis).
+
+-   Arguments related to configuring the sleep detection algorithm:
+    `anglethreshold`, `timethreshold`, `HASPT.algo`, `HASIB.algo`,
+    `Sadeh_axis`, and `HASPT.ignore.invalid`.
+-   `ignorenonwear` if set to TRUE then ignore detected monitor non-wear
+    periods in the detection of sustained inactivity bouts to avoid
+    confusion between monitor non-wear time.
+-   If you want to create a visualisation of how sleep period time and
+    sustained inactivity bouts match throughout a day then consider
+    arguments `do.visual`, `outliers.only`, and `criterror`.
+-   If you want to exclude the first and last night from the sleep
+    analysis then used `excludefirstlast`.
+-   `def.noc.sleep` specifies how the sleep period time window should be
+    estimated if no sleeplog is used.
+-   `includenightcrit` Minimum number of valid hours per night (24 hour
+    window between noon and noon or 6pm-6pm).
+-   `data_cleaning_file` to ginore specific nights for specific
+    individuals, see also section [Data cleaning
+    file](#Data_cleaning_file).
+-   If you want the sleep analysis to be guided by a sleeplog (diary)
+    then check out arguments `loglocation` which specifies the location
+    of the spreadsheet (csv) with sleep log information. Further, use
+    arguments `colid`, `coln1`, `nnights`, and `sleeplogsep` to specify
+    the details of your sleep log structure.
 
 GGIR facilitates two possible sleeplog file structures:
 
 #### Basic sleep log
 
 Example of a basic sleeplog:
 
-ID | onset_N1 | wakeup_N1 | onset_N2 | wakeup_N2 | onset_N3 | wakeup_N3 | onset_N4 | ...
----|---|---|---|---|---|---|---|---
-123 | 22:00:00 | 09:00:00 | 21:00:00 | 08:00:00 | 23:45:00 | 06:30:00 | 00:00:00 | ...
-345 | 21:55:00 | 08:47:00 |  |  | 23:45:00 | 06:30:00 | 00:00:00 | ...
-
-- One column for participant id, this does not have to be the first column. Specify which column it is with  argument `colid`.
-- Alternatingly one column for onset time and one column for waking time. Specify which column is the column for the first night by argument `coln1`, in the above example `coln1=2`.
-- Timestamps are to be stored without date as in hh:mm:ss with hour values ranging between 0 and 23 (not 24). If onset corresponds to lights out or intention to fall asleep, then it specify `sleepwindowType = "TimeInBed"`.
-- There can be multiple sleeplogs in the same spreadsheet. Each row representing a single recording.
-- First row: The first row of the spreadsheet needs to be filled with column names. For the basic sleep log format it does not matter what these column names are.
-
+| ID  | onse t_N1 | w akeu p_N1 | onse t_N2 | w akeu p_N2 | onse t_N3 | w akeu p_N3 | onse t_N4 | ... |
```

Comment:
Something went wrong there are various space that in the column names that should not be there.

Reply:
Revised